subworkflows/local/contaminant_filter/contaminant_filter.nf

```diff
@@ -0,0 +1,261 @@
+//
```

Comment:
This file should be labeled main.nf since it is already inside a folder labeled contaminant_filter